packages/vite-plugin-checker/src/logger.ts

```diff
@@ -262,8 +230,8 @@ export function normalizeLspDiagnostic({
   fileText: string
 }): NormalizedDiagnostic {
   let level = DiagnosticLevel.Error
-  const loc = lspRange2Location(diagnostic.range)
-  const codeFrame = codeFrameColumns(fileText, loc)
+  const loc = tsLocationToBabelLocation(diagnostic.range)
+  const codeFrame = createFrame(fileText, loc)
```

Comment:
vue-vls is now using `createFrame` like all other checkers (change: colors and indention)